lib/helper.coffee

```diff
@@ -0,0 +1,91 @@
+ChildProcess = require 'child_process'
+Path = require 'path'
+FS = require 'fs'
+{XRegExp} = require 'xregexp'
+
+class Helpers
+  @findFile: (directory, fileName) ->
+    climb = directory.split(Path.sep)
+    while climb.length
+      target = Path.join(climb.join(Path.sep), fileName)
+      if FS.existsSync(target)
+        return target
+      climb.pop()
+  @exec: (command, cwd, stdin) ->
+    executionTime = 5000
+    return new Promise (resolve, reject) ->
+      timeout = null
+      spawnedProcess = ChildProcess.exec "#{command.join(' ')}", {cwd}, (err, stdout, stderr)->
```

Comment:
Are you sure that no linters will ever run into any of the bugs that `BufferedProcess` (or whatever it's called) was designed to fix?